src/mlpack/bindings/go/print_defn_input.hpp

```diff
@@ -109,7 +109,7 @@ void PrintDefnInput(
  * @param * (output) Unused parameter.
  */
 template<typename T>
-void PrintDefnInput(const util::ParamData& d,
+void PrintDefnInput(util::ParamData& d,
                            const void* /* input */,
                            void* /* output */)
```

Comment:
```suggestion
void PrintDefnInput(util::ParamData& d,
                     const void* /* input */,
                     void* /* output */)
```
Might as well fix this one too. :)